manifests/resource/mailhost.pp

```diff
@@ -102,10 +153,10 @@
   Optional[String] $ssl_trusted_cert             = undef,
   Optional[Integer] $ssl_verify_depth            = undef,
   Enum['on', 'off', 'only'] $starttls            = 'off',
-  $protocol                                      = undef,
+  Optional[Enum['imap', 'pop3', 'smtp']] $protocol = undef,
```

Comment:
These are the only values allowed. Previously only documented but https://nginx.org/en/docs/mail/ngx_mail_core_module.html#protocol agrees.